Api/Src/Models/User.ts

```diff
@@ -1,15 +1,17 @@
 import * as mongoose from 'mongoose';
 import {Grade} from './Grade';
+import * as Joi from "joi";
+const passwordComplexity = require("joi-password-complexity");
```

Comment:
Is there a reason why it is not ES6 import? There is more of that in your changes.

Reply:
There are some problems if I import it in the standard way - doesn't work with Joi.

Reply:
hmm, alright 👍